### ISSUE TITLE
Update README (as of v0.0.5)

### ISSUE THREAD

**Issue by lsrdg** [lsrdg/Tatoeba-query#35]
The upcoming v0.0.5 release is an important milestone to the project and the
README needs to be improved. 

The README needs to be more concise about: 

- what the project does (and what it doesn't), 
- what are the short and long time goals,
- where is the development happening (#18 )

Try to be a bit more clear about the commands. Find a better way to present the
features.

The README will still be the "docs"
([v0.0.6](https://github.com/lsrdg/tatoeba-karini/milestone/3) will be shiped
with a better documentation). Despite of that, the "how to"/"getting started"
needs to be improved.

The commands can be divided into three groups:

1) - those depending on internet connections
      - (`-b`) open the browser searching for a term in/to the given languages
      - (`-i`) like `-b` but searching by sentence's id
      - (`-r`) request/scrap the website

2) - those that perform locally
      - (`f`) find sentence, in a given language, containing term in the target
      language
      - (`s`) like `-f`, plus finds translations to the found sentence
3) - helpers (to provide a better (group(s) 1 and/or 2)
      - (`-d`) download and uncompress files
      - (`-l`) given a language, return its abbreviation

---

Points to be reviewed:

- README's structure
- Requirements
- Getting started
- Current stage X Notes
- TODO becomes something else (roadmap, maybe?)
- Acknowledge contributions
- Actually add a license to the code (creative content's license is noted)

**Comment by lsrdg** [comment]
#18 should be listed above as well.

---
EDIT: add build status badge:
```
[![Build Status](https://travis-ci.org/lsrdg/tatoeba-karini.svg?branch=master)](https://travis-ci.org/lsrdg/tatoeba-karini)
```

**Comment by lsrdg** [comment]
Closed in #36 .
